core/store/orm/schema.go

```diff
@@ -25,16 +25,18 @@ type ConfigSchema struct {
 	DefaultHTTPTimeout              models.Duration `env:"DEFAULT_HTTP_TIMEOUT" default:"15s"`
 	Dev                             bool            `env:"CHAINLINK_DEV" default:"false"`
 	EnableExperimentalAdapters      bool            `env:"ENABLE_EXPERIMENTAL_ADAPTERS" default:"false"`
+	EnableBulletproofTxManager      bool            `env:"ENABLE_BULLETPROOF_TX_MANAGER" default:"false"`
```

Comment:
Is this intentionally different from MIN_INCOMING_CONFIRMATIONS?

Reply:
Which one.. do you mean `EthFinalityDepth`?

Yes, it is intentionally different.